ciri/consensus/ledger_validator/tests/BUILD

```diff
@@ -20,18 +20,10 @@ cc_test(
         "//ciri/consensus/test_utils",
         "//ciri/consensus/transaction_solidifier",
         "//common/helpers:digest",
-        "//common/storage/sql/sqlite3:sqlite3_storage",
-        "//common/storage/tests/helpers",
+        "//common/storage/sql/sqlite3:storage_sqlite3",
```

Comment:
shouldn't we select here based on `config_settings` ?

Reply:
Definitely